sql/core/src/test/scala/org/apache/spark/sql/ArrowConvertersSuite.scala

```diff
@@ -170,12 +170,12 @@ class ArrowConvertersSuite extends SharedSQLContext {
     val jsonSchema = jsonReader.start()
     Validator.compareSchemas(arrowSchema, jsonSchema)
 
-    val arrowPayload = df.collectAsArrow(Some(converter))
+    // TODO: repartition because can only validate one batch
+    val arrowPayload = df.repartition(1).collectAsArrow(Some(converter))
```

Comment:
These tests fail if using more than 1 partition because it will produce more than 1 ArrowRecordBatches and I think `VectorLoader` will just load 1 batch.  Is there a way to combine batches or load more than 1 in Java?